lua/nightfox/extra/nushell.lua

```diff
@@ -0,0 +1,56 @@
+local template = require("nightfox.util.template")
+
+local M = {}
+
+function M.generate(spec)
+  local content = [[
+let nightfox_theme = {
+   binary: "${syntax.number}"
+   block: "${syntax.bracket}"
+   bool: "${syntax.const}"
+   cellpath: "${palette.white}"
```

Comment:
I would be careful with setting either while or black by themselves. The issue is that for light themes like dayfox these colors will be very subtle. I would suggest that you append to the `spec` table with the following:

```lua
local P = spec.palette
spec.harsh = P.meta.light and P.black or P.white
spec.subtle = P.meta.light and P.white or P.black
```

This would then allow you to call `cellpath: ${harsh}`. This would be correct for both dark and light themes.